boards/forms.py

```diff
@@ -69,11 +69,14 @@ def invite(self, host, username, board):
         new_referral.user = user
         new_referral.save()
 
-    def remove_member(self, to_remove):
+    def remove_members(self, to_remove):
```

Comment:
its much better if you get all of the objects using id__in=[1,2,3] e.g MyObjects.objects.filter(id__in=[1,2,3])
then iterate the objects and delete one by one.